php/check.php

```diff
@@ -0,0 +1,6 @@
+<?php
+if($_POST){
+  header('Location: ./mail.php');
+}
+session_start();
+$_SESSION = $_POST;
```

Comment:
__[code_sniffer]__
* Expected 1 newline at end of file; 0 found

<!-- Comment by [SideCI](https://sideci.com) -->